Add ${PREFIX}/etc as an acceptable location for emboss.default.  (This should
be incorporated into the next version of EMBOSS.)

The port installs ${PREFIX}/etc/emboss.default.sample

-- Johann <johann@egenetics.com>


--- ajax/ajnam.c.orig	Wed Aug  2 10:18:59 2000
+++ ajax/ajnam.c	Thu Aug 10 10:06:20 2000
@@ -982,6 +982,13 @@
     prefixRootFile = fopen (ajStrStr(namRootStr), "r");
   }
 
+  /* look for $(PREFIX)_ROOT/../emboss.default */
+
+  if (!prefixRootFile) {
+    ajFmtPrintS (&namRootStr, "%s/etc/%s.default", prefixRoot, prefix);
+    prefixRootFile = fopen (ajStrStr(namRootStr), "r");
+  }
+
   if (namFileOrig)
     (void) ajStrAppC (&namFileOrig, ", ");
   (void) ajStrApp (&namFileOrig, namRootStr);
